File: app/src/main/java/com/platdmit/simplecloudmanager/data/api/models/ApiV4.java
package com.platdmit.simplecloudmanager.data.api.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ApiV4 {
    @SerializedName("ip_address")
    @Expose
    private String ipAddress;
    @SerializedName("netmask")
    @Expose
    private String netmask;
    @SerializedName("gateway")
    @Expose
    private String gateway;

    public String getIpAddress() {
        return ipAddress;
    }

    public void setIpAddress(String ipAddress) {
        this.ipAddress = ipAddress;
    }

    public String getNetmask() {
        return netmask;
    }

    public void setNetmask(String netmask) {
        this.netmask = netmask;
    }

    public String getGateway() {
        return gateway;
    }

    public void setGateway(String gateway) {
        this.gateway = gateway;
    }
}

File: data/src/main/java/com/platdmit/data/retrofit/models/ApiV4.kt
package com.platdmit.data.retrofit.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ApiV4 (
        @SerializedName("ip_address")
        @Expose

        val ipAddress: String,
        @SerializedName("netmask")
        @Expose

        val netmask: String,
        @SerializedName("gateway")
        @Expose
        val gateway: String
)